README.rst

```diff
@@ -0,0 +1,40 @@
+=======
+RavenPy
+=======
+
+
+.. image:: https://img.shields.io/pypi/v/ravenpy.svg
+        :target: https://pypi.python.org/pypi/ravenpy
+
+.. image:: https://img.shields.io/travis/CSHS-CWRA/ravenpy.svg
+        :target: https://travis-ci.com/CSHS-CWRA/ravenpy
+
+.. image:: https://readthedocs.org/projects/ravenpy/badge/?version=latest
+        :target: https://ravenpy.readthedocs.io/en/latest/?badge=latest
+        :alt: Documentation Status
+
+.. image:: https://pyup.io/repos/github/CSHS-CWRA/ravenpy/shield.svg
+        :target: https://pyup.io/repos/github/CSHS-CWRA/ravenpy/
+        :alt: Updates
+
+
+
+A Python wrapper to setup and run the hydrologic modelling framework Raven.
+
+
+* Free software: MIT license
+* Documentation: https://ravenpy.readthedocs.io.
+
+
+Features
+--------
+
+* TODO
+
+Credits
+-------
+
+This package was created with Cookiecutter_ and the `audreyr/cookiecutter-pypackage`_ project template.
```

Comment:
We probably should reference our cookiecutter instead here.

Reply:
Done!